### ISSUE TITLE
ORDS error message when only specifing IP/ords - But only after startup, once primed works fine

### ISSUE THREAD

**Issue by GasparYYC** [OraOpenSource/node4ords#23]
Hi
  The IP address of my server is xxx.xxx.xxx.xxx. If I attempt to go to: http://xxx.xxx.xxx.xxx/ords (excluding /f?p=4550 from the URL), the page simply outputs: 

ORACLE REST DATA SERVICES
503 Service Unavailable

The connection pool named: apex is not correctly configured, due to the following error(s): Unable to start the Universal Connection Pool: oracle.ucp.UniversalConnectionPoolException: Error during pool creation in Universal Connection Pool Manager: oracle.ucp.UniversalConnectionPoolException: Universal Connection Pool already exists in the Universal Connection Pool Manager. Universal Connection Pool cannot be added to the Universal Connection Pool Manager

I noticed errors in the startup log:

```
# systemctl status node4ords -l
  node4ords.service - Node4Ords - proxy to APEX using ORDS on Tomcat
   Loaded: loaded (/etc/systemd/system/node4ords.service; enabled; vendor preset: disabled)
   Active: active (running) since Wed 2015-12-30 15:40:42 CST; 43min ago
 Main PID: 1078 (node4ords)
   CGroup: /system.slice/node4ords.service
           ├─1078 /bin/bash /usr/local/bin/node4ords
           └─1085 node /opt/node4ords/app.js

Dec 30 16:15:02 cm1 node4ords[1078]: syscall: 'connect' }
Dec 30 16:15:45 cm1 node4ords[1078]: proxy.web error:  { [Error: socket hang up] code: 'ECONNRESET' }
Dec 30 16:16:29 cm1 node4ords[1078]: proxy.web error:  { [Error: socket hang up] code: 'ECONNRESET' }
Dec 30 16:17:25 cm1 node4ords[1078]: proxy.web error:  { [Error: socket hang up] code: 'ECONNRESET' }
Dec 30 16:18:01 cm1 node4ords[1078]: proxy.web error:  { [Error: socket hang up] code: 'ECONNRESET' }
Dec 30 16:18:07 cm1 node4ords[1078]: proxy.web error:  { [Error: socket hang up] code: 'ECONNRESET' }
Dec 30 16:18:17 cm1 node4ords[1078]: proxy.web error:  { [Error: socket hang up] code: 'ECONNRESET' }
Dec 30 16:18:30 cm1 node4ords[1078]: proxy.web error:  { [Error: socket hang up] code: 'ECONNRESET' }
Dec 30 16:18:32 cm1 node4ords[1078]: proxy.web error:  { [Error: socket hang up] code: 'ECONNRESET' }
Dec 30 16:20:34 cm1 node4ords[1078]: proxy.web error:  { [Error: socket hang up] code: 'ECONNRESET' }
```

The interesting part is that if I "kickstart it" but including the "f?p=4550" part of the arguments then I see the login screen (meaning worked fine), after this point it woks fine if I only do: xxx.xxx.xxx.xxx/ords  

In summary: it works after being primed by providing the argument.

Thanks
